.github/workflows/reusable_basic.yml

```diff
@@ -247,6 +247,13 @@ jobs:
             shared_library: 'ON'
             level_zero_provider: 'OFF'
             cuda_provider: 'OFF'
+          - os: 'windows-2022'
+            build_type: Release
+            compiler: {c: cl, cxx: cl}
+            shared_library: 'ON'
+            level_zero_provider: 'ON'
+            cuda_provider: 'ON'
+            umfd_lib: 'ON'
```

Comment:
Please, remove this and add flag `umfd_lib: 'ON'` to the up:
```

          - os: 'windows-2022'
            build_type: Release
            compiler: {c: cl, cxx: cl}
            shared_library: 'ON'
            level_zero_provider: 'ON'
            cuda_provider: 'ON'
```
This job is redundant with the OS matrix, so give him a new life :)